flight/targets/coptercontrol/fw/pios_board.c

```diff
@@ -140,6 +140,10 @@ void PIOS_Board_Init(void) {
 	/* Mount all filesystems */
 	PIOS_FLASHFS_Logfs_Init(&pios_uavo_settings_fs_id, &flashfs_settings_cfg, FLASH_PARTITION_LABEL_SETTINGS);
 
+#if defined(ERASE_FLASH)
+	PIOS_FLASHFS_Format(pios_uavo_settings_fs_id);
+#endif
+
```

Comment:
So would you prefer in this PR I rip it out from F3/F4 and then have it -only- in CC3D and pipxtreme?